selenium_test.py

```diff
@@ -0,0 +1,222 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+import pandas as pd
+import time
+
+KEYWORDS_FILE = 'textfile.txt'
+RANKED_OUTPUT_FILE = 'Output.csv'
+START_YEAR = "1917"
+END_YEAR = "1971"
+
+
+class resultLists:
+    title_list = []
+    description_list = []
+    category_list = []
+    keyword_list = []
+    link_list = []
```

Comment:
No, that's not any better. Imagine if you had a system that dealt with students - would you have a `first_name_list`, `last_name_list`, `email_list`, etc? No!

Rather, you would have a `Student` class, and then a list of Students (or maybe a map with the email or ID as the key). Same thing here - have a `Link` or `Article` class, then put them into containers.